src/test/java/unit/Group.java

```diff
@@ -106,6 +106,28 @@ void listDescribeUsersReflectCreateDelete() throws IOException {
     matchedGroup =
         groupList.stream().filter(listedGroup -> listedGroup.name.equals(name)).findAny();
     assertTrue(matchedGroup.isEmpty(), "group does not appear in list after delete");
+
+    // create another group
+    String name1 = SamGroups.randomGroupName();
+    TestCommand.runAndParseCommandExpectSuccess(
+        UFGroup.class, "group", "create", "--name=" + name1);
+
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list");
+
+    // use regular expression testing the table format and content inside
+    assertEquals(0, cmd.exitCode, "group list returned successfully");
+    String[] rows = cmd.stdOut.split("\\r?\\n");
+    String[] rowHead = rows[0].split("\\s+");
+    assertEquals(rowHead[0].trim().replace("\r", ""), "EMAIL");
+    assertEquals(rowHead[1].trim().replace("\r", ""), "MEMBERS");
+    assertEquals(rowHead[2].trim().replace("\r", ""), "POLICIES");
+
+    for (int i = 1; i < rows.length; i = i + 1) {
+      String[] rowi = rows[i].split("\\s+");
+      assertTrue(rowi[0].matches("^[a-zA-Z0-9_-]+@[a-zA-Z0-9_-]+(\\.[a-zA-Z0-9_-]+)+$"));
+      assertTrue(rowi[1].matches("[0-9]+"));
+      assertTrue(Arrays.asList("[ADMIN]", "[MEMBER]").contains(rowi[2]));
```

Comment:
Have we found a way to display more than one policy? That would break this check.